src/api/AlertIOS.js

```diff
@@ -35,11 +35,6 @@ class AlertIOS {
    * ```
    */
   static alert(title, message, callbackOrButtons, type) {
-    if (typeof type !== 'undefined') {
-      console.warn('AlertIOS.alert() with a 4th "type" parameter is deprecated and will be removed. Use AlertIOS.prompt() instead.');
-        this.prompt(title, message, callbackOrButtons, type);
```

Comment:
I think the main reason was to clean it up and get rid of deprication warnings. As that's now going to be a new PR, i'll add it back!